packages/core/src/render3/di.ts

```diff
@@ -317,18 +317,18 @@ class ElementRef implements viewEngine.ElementRef {
  */
 export function getOrCreateTemplateRef<T>(di: LNodeInjector): viewEngine.TemplateRef<T> {
   const data = (di.node as LContainer).data;
-  if (data === null || data.template === null) {
+  /*if (data === null || data.template === null) {
```

Comment:
did you intend to comment this out?

Reply:
Yes, actually to delete it. We've got many cases where `<ng-template></ng-template>` will have no content (basically all use-cases where it is used to mark an insertion point) and IMO it is better to have `null` as a template function instead of having a "dummy" (empty) template function. Also the error message was misleading since we can have a `<ng-template>` without any directive "sitting on top".

We could still leave a check for `data` not being `null`, though....